src/Diagnostics.CompilerHost/Startup.cs

```diff
@@ -150,8 +150,12 @@ public void ConfigureServices(IServiceCollection services)
                                         AddAuthenticationSchemes(CertificateAuthDefaults.AuthenticationScheme, "AzureAd")
                                         .Build();
             });
-            // Enable app insights telemetry
-            services.AddApplicationInsightsTelemetry();
+
+            if (!Environment.IsDevelopment())
+            {
```

Comment:
While this looks good to me, I am not too sure if someone is relying on App Insights Telemetry from the dev environement. @rekhaswaminathan do you know if AI is used for DEV environment at all?

Reply:
We don't rely on App Insights telemetry for dev environment

Reply:
Thank you for confirmation. I am resolving this comment.